modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -6689,6 +6690,7 @@ enter-one-screen-name-per-line-to-reserve-the-screen-name=Enter one screen name
 enter-one-url-per-line=Enter one URL <strong>per line</strong>.
 enter-one-user-email-address-per-line-to-reserve-the-user-email-address=Enter one user email address <strong>per line</strong> to reserve the user email address.
 enter-one-value-per-line=Enter one value per line.
+enter-path=Enter path
```

Comment:
@lhine Since this is a title, you should follow the rules of https://capitalizemytitle.com/. So the value here will be `Enter Path`.